File: archive/j/javascript/fizz-buzz.js
function fizzbuzz(num){
    for(let i=1; i <= num; i++){
      if(i % 15 == 0){
        console.log("fizzbuzz");
      }
      else if(i % 5 == 0){
        console.log("buzz");
      }
      else if(i % 3 == 0){
        console.log("fizz");
      }
      else console.log(i);
   }
  }
  
fizzbuzz(100);

File: archive/j/java/FizzBuzz.java
public class FizzBuzz {
    public static void main(String[] args) {
        for (int i = 1; i < 101; i++) {
            String output = "";
            if (i % 3 == 0) {
                output += "Fizz";
            }
            if (i % 5 == 0) {
                output += "Buzz";
            }
            if (output.isEmpty()) {
                output += i;
            }
            System.out.println(output);
        }
    }
}
